### ISSUE TITLE
Uncaught Error: Cannot find module 'stream' with node stubs 1.1.0

### ISSUE THREAD

**Issue by WayneUong** [meteor/meteor#11974]
Using `vue-server-renderer` with `meteor-node-stubs` 1.1.0 produces this error on the client side:

```
Uncaught Error: Cannot find module 'stream'                       modules-runtime.js?hash=834e737986edfacdf20bbc352d7f3f62e50d0b47:232 
    at makeMissingError (modules-runtime.js?hash=834e737986edfacdf20bbc352d7f3f62e50d0b47:232:12)
    at Module.require (modules-runtime.js?hash=834e737986edfacdf20bbc352d7f3f62e50d0b47:251:17)
    at require (modules-runtime.js?hash=834e737986edfacdf20bbc352d7f3f62e50d0b47:268:21)
    at module (modules.js?hash=3463d39b3e434e2405c007b94255528fdcc0a720:3612:14)
    at fileEvaluate (modules-runtime.js?hash=834e737986edfacdf20bbc352d7f3f62e50d0b47:346:7)
    at Module.require (modules-runtime.js?hash=834e737986edfacdf20bbc352d7f3f62e50d0b47:248:14)
    at require (modules-runtime.js?hash=834e737986edfacdf20bbc352d7f3f62e50d0b47:268:21)
    at module (modules.js?hash=3463d39b3e434e2405c007b94255528fdcc0a720:1104:20)
    at fileEvaluate (modules-runtime.js?hash=834e737986edfacdf20bbc352d7f3f62e50d0b47:346:7)
    at Module.require (modules-runtime.js?hash=834e737986edfacdf20bbc352d7f3f62e50d0b47:248:14)
```

Reverting back to node stubs 1.0.3 will fix this.

Reproducing error: https://github.com/WayneUong/meteor-node-stubs-error

**Comment by MikeYuanMY** [comment]
We also ran into this issue 

Seems like the vue package is calling the stream package here 
`vuejs/vue/packages/vue-server-renderer/build.dev.js` 

```javascript
 /**
 * Original RenderStream implementation by Sasha Aickin (@aickin)
 * Licensed under the Apache License, Version 2.0
 * http://www.apache.org/licenses/LICENSE-2.0
 *
 * Modified by Evan You (@yyx990803)
 */

var stream = require('stream');
```

**Comment by pfafman** [comment]
Having similar issues here with a number of require statements for npm packages through out the app.  Reverting back to meteor-node-stubs@1.0.0 fixes the errors.  The errors are also there in the new 1.2.0 version.

**Comment by StorytellerCZ** [comment]
Any idea which of the packaged that are part of the node stubs is causing this?

**Comment by alexanderwanyoike** [comment]
Quite a few missing dependencies with meteor-node-stubs@1.2.0

```
warning "meteor-node-stubs@1.2.0" is missing a bundled dependency "assert". This should be reported to the package maintainer.
warning "meteor-node-stubs@1.2.0" is missing a bundled dependency "browserify-zlib". This should be reported to the package maintainer.
warning "meteor-node-stubs@1.2.0" is missing a bundled dependency "buffer". This should be reported to the package maintainer.
warning "meteor-node-stubs@1.2.0" is missing a bundled dependency "console-browserify". This should be reported to the package maintainer.
warning "meteor-node-stubs@1.2.0" is missing a bundled dependency "constants-browserify". This should be reported to the package maintainer.
warning "meteor-node-stubs@1.2.0" is missing a bundled dependency "crypto-browserify". This should be reported to the package maintainer.
warning "meteor-node-stubs@1.2.0" is missing a bundled dependency "domain-browser". This should be reported to the package maintainer.
warning "meteor-node-stubs@1.2.0" is missing a bundled dependency "events". This should be reported to the package maintainer.
warning "meteor-node-stubs@1.2.0" is missing a bundled dependency "https-browserify". This should be reported to the package maintainer.
warning "meteor-node-stubs@1.2.0" is missing a bundled dependency "os-browserify". This should be reported to the package maintainer.
warning "meteor-node-stubs@1.2.0" is missing a bundled dependency "path-browserify". This should be reported to the package maintainer.
warning "meteor-node-stubs@1.2.0" is missing a bundled dependency "process". This should be reported to the package maintainer.
warning "meteor-node-stubs@1.2.0" is missing a bundled dependency "punycode". This should be reported to the package maintainer.
warning "meteor-node-stubs@1.2.0" is missing a bundled dependency "querystring-es3". This should be reported to the package maintainer.
warning "meteor-node-stubs@1.2.0" is missing a bundled dependency "readable-stream". This should be reported to the package maintainer.
warning "meteor-node-stubs@1.2.0" is missing a bundled dependency "stream-browserify". This should be reported to the package maintainer.
warning "meteor-node-stubs@1.2.0" is missing a bundled dependency "stream-http". This should be reported to the package maintainer.
warning "meteor-node-stubs@1.2.0" is missing a bundled dependency "string_decoder". This should be reported to the package maintainer.
warning "meteor-node-stubs@1.2.0" is missing a bundled dependency "timers-browserify". This should be reported to the package maintainer.
warning "meteor-node-stubs@1.2.0" is missing a bundled dependency "tty-browserify". This should be reported to the package maintainer.
warning "meteor-node-stubs@1.2.0" is missing a bundled dependency "url". This should be reported to the package maintainer.
warning "meteor-node-stubs@1.2.0" is missing a bundled dependency "util". This should be reported to the package maintainer.
warning "meteor-node-stubs@1.2.0" is missing a bundled dependency "vm-browserify". This should be reported to the package maintainer.

```

**Comment by fredmaiaarantes** [comment]
Thanks for reporting it! I believe we fixed the missing dependencies in version `1.2.1`. 

Could you please give it a try and let me know if it works?

**Comment by pfafman** [comment]
Is working here with meteor 2.6.1 and node stubs 1.2.1.  Thanks.

**Comment by fredmaiaarantes** [comment]
I'm closing this issue as it worked on the tests I made and also for @pfafman.
You can reopen it if you still have this issue.

**Comment by pfafman** [comment]
Version 1.2.2 is back to giving errors about not being able to find npm packages!

**Comment by denihs** [comment]
Hey @pfafman, could you update to version 1.2.3 and test it again?

**Comment by pfafman** [comment]
That fixed it.  Thanks

**Comment by alimgafar** [comment]
@denihs For clarity, could you summarize what happened and what the fix was?

**Comment by denihs** [comment]
I ran `npm install` before publishing the new version, and it fixed it.

**Comment by valerymagadeeva** [comment]
Hi! The same issue with the package 1.2.5 and higher. 1.2.3 is still correct
